### PR TITLE
fix: weekly dependabot updates - don't drown devs

### PR DESCRIPTION
**Pull request by ahammond**
Fixes #

### CODE REVIEW

**Comment by ashby2** [comment]
@ahammond what was the default for dependabot before this change to weekly?

**Review by ashby2 — APPROVED** [review]
looks good!

File: test/clickup-ts.test.ts [hunk 1]
```diff
@@ -8,8 +8,20 @@ describe('ClickUpTypeScriptProject', () => {
...
+        expect(synth['codecov.yml']).toMatchSnapshot();
+      });
+      test('package.json', () => {
+        expect(synth['package.json']).toMatchSnapshot();
+      });
+      test('upgrade-main', () => {
+        expect(synth['.github/workflows/upgrade-main.yml']).toMatchSnapshot();
+      });
+      test('mergify', () => {
+        expect(synth['mergfiy.yml']).toMatchSnapshot();
```
Comment (Justinon) [inline test/clickup-ts.test.ts:23]:
```suggestion
        expect(synth['mergify.yml']).toMatchSnapshot();
```
Probably what you wanted?

Comment (Justinon) [reply test/clickup-ts.test.ts:23]: Maybe even `.mergify.yml` since that's the name of the file at the root of the repo?

**Pull request merged**